DonjinKrawler-Client/src/main/java/donjinkrawler/items/Weapon.java

```diff
@@ -37,6 +38,11 @@ public String getDescription() {
                 + "Speed: " + this.getSpeed();
     }
 
+    @Override
+    public void accept(ItemVisitor v) {
```

Comment:
From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Use intention revealing names.`